packages/components/src/ui/ariakit-tooltip/types.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import type { TooltipStateProps } from 'ariakit/tooltip';
+
+export type ToolTipProps = Pick< TooltipStateProps, 'placement' > & {
+	/**
+	 * The anchor for the tooltip. Accepts only one child element.
+	 */
+	children: React.ReactElement | string;
+	/**
+	 * The amount of time in milliseconds to wait before showing the tooltip.
+	 *
+	 * @default 700
+	 */
+	delay?: number;
+	/**
+	 * Where the tooltip should be positioned relative to its parent.
+	 *
+	 * @default
+	 */
+	placement?: any; //Placement | undefinded;
+	/**
+	 * _Note: this prop is deprecated. Please use the `placement` prop instead._
+	 *
+	 * Legacy way of specifying the tooltip's position relative to its parent.
+	 *
+	 * @default top center
```

Comment:
The `position` prop doesn't have a default value anymore (since it's being replaced by the `placement` prop, which has its own equivalent default value). We should therefore remove this `@default` tag in the JSDocs